setup.py

```diff
@@ -80,7 +81,54 @@ def numpy_include_dir():
     return np.get_include()
 
 
-def extension_modules():
+def _sha256_hash(filename, blocksize=65536):
+    """Hash the contents of an open file handle with SHA256"""
+    hash_obj = hashlib.sha256()
+
+    with open(filename, "rb") as file_obj:
```

Comment:
H/T to https://github.com/pypa/auditwheel/blob/d5cd0f670eecf9a94be67911e18d3c2dea8ffc55/auditwheel/hashfile.py